packages/features/portfolio/src/containers/portfolio-holdings/use-portfolio-holdings.tsx

```diff
@@ -114,6 +115,44 @@ export function usePortfolioHoldings() {
     }
   }, [expandedRows, setExpandedRows, Columns]);
 
+  const portfolioHoldingsData =
+    toggleOption === 0 && !isBlocked && groupedRows.length > 0
+      ? groupedHoldings
+      : detailedHoldings;
+
+  function insertDebtDivider(arr) {
```

Comment:
Can you move this function declaration outside of the hook? If it is declared inside it will be recreated on every render